src/views/pages/profiles/LoginPage.vue

```diff
@@ -9,6 +9,12 @@
                 <img class="language_icon" :src="require('@/views/resources/img/login/language.svg')" alt="" />
                 <LanguageSelector />
             </div>
+
+            <div v-if="versionCheckerObject && !latestVersionInUse" class="update-box" role="alert">
+                <b syle="">{{ $t('new_version_available') }}</b>
```

Comment:
Can you please resolve this issue from Sonar?

> Replace this `<b>` tag by `<strong>`.

```suggestion
                <strong>{{ $t('new_version_available') }}</strong>
```